### PR TITLE
Issue/auth flow

### LINKED ISSUE #53: Log in screen api calls
Goes hand in hand with #46

### PR DESCRIPTION
**Pull request by kennygperez**
Resolves #53 #49

### CODE REVIEW

**Review by Goff-Davis — CHANGES_REQUESTED** [review]
When I run the backend I get:

```
ERROR in src/handlers/db.ts:25:4
[0] TS2345: Argument of type '{ canvasId: number; fullName: string; }' is not assignable to parameter of type 'UserCreationAttributes'.
[0]   Object literal may only specify known properties, and 'fullName' does not exist in type 'UserCreationAttributes'.
[0]     23 |            await User.create({
[0]     24 |                    canvasId: 1,
[0]   > 25 |                    fullName: 'Mock',
[0]        |                    ^^^^^^^^^^^^^^^^
[0]     26 |            });
```

The mobile seems to work but I am unable to sign in since the backend is throwing errors.

**Comment by kennygperez** [comment]
@Goff-Davis that's the thing Toan and I were talking about

That's mock data that is messing with real data from Canvas

Im going to push the script with that section removed

**Comment by kennygperez** [comment]
I also added info to the README

**Review by weining-li — APPROVED** [review]

File: backend/src/util/token.ts
Comment (kennygperez) [reply backend/src/util/token.ts:4]: ill add a TODO to this

Comment (kennygperez) [reply backend/src/util/token.ts:5]: Pushed todo

Comment (Goff-Davis) [reply backend/src/util/token.ts:5]: Please put this in the backend `.env`. It's prob a good idea to rebase the commits after too so it isn't in the history. We can't be commiting secrets.

**Review by Goff-Davis — CHANGES_REQUESTED** [review]

**Review by Goff-Davis — APPROVED** [review]
Approved as long as the private key gets removed and rebased after merge

**Pull request merged**
